### ISSUE TITLE
Readonly Attribute

### ISSUE THREAD

**Issue by LorenDorez** [vsn4ik/bootstrap-checkbox#20]
The Checkbox Plug isnt respecting the ReadOnly attribute.

I need to use ReadOnly so that the control still postback to the server, because disabled fiedls dont post

**Comment by LorenDorez** [comment]
I recommend changing the Disabled check to as follows

FROM
 if (this.element.disabled) {

TO
if (this.element.disabled || this.element.readOnly) {

**Comment by vsn4ik** [comment]
Readonly not supported for `input[type="checkbox"]` by default.

For readonly use
```html
<input type="hidden" name="..." value="0|1">
<p>This is readonly</p>
```

Also see: http://getbootstrap.com/css/#forms-controls-static

**Comment by vsn4ik** [comment]
Fixed in `v1.5.0`.
Thanks @LorenDorez!
